ush/python/pygfs/task/marine_letkf.py

```diff
@@ -43,6 +64,53 @@ def initialize(self):
 
         logger.info("initialize")
 
+        # make directories and stage ensemble background files
+        letkf_stage_list = parse_j2yaml(self.task_config.MARINE_LETKF_STAGE_YAML_TMPL, self.task_config)
+        FileHandler(letkf_stage_list).sync()
+        letkf_stage_fix_list = parse_j2yaml(self.task_config.SOCA_FIX_STAGE_YAML_TMPL, self.task_config)
+        FileHandler(letkf_stage_fix_list).sync()
+
+        # TODO(AFE): probably needs to be jinjafied
+        obs_list = YAMLFile(self.task_config.OBS_YAML)
```

Comment:
```suggestion
        obs_list = parse_j2yaml(self.task_config.OBS_YAML, self.task_config)
```
So when the yaml is jinjafied, there is no change needed in the python code.

Reply:
done